### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Ellyboi**
**Hello @CODE REVIEWER ❤️ 💌 , Look at my project and give your criticism either positive or negative, though i believe it will be positive 😆**  

- Implemented the design from the sneak peek wireframes.
-I focused more on making sure everything functions first

pictoria representations below:

today

today1

today2

### CODE REVIEW

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Ellyboi,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice views ✔️
- Descriptive PR. ✔️

## Required Changes ♻️
_Check the comments below._

- [ ] Hi, when trying to create a comment for a post, the database throws this error 
```
rails aborted!
ActiveRecord::NotNullViolation: PG::NotNullViolation: ERROR:  null value in column "post_id" of relation "comments" violates not-null constraint
DETAIL:  Failing row contains (1, 4, null, Hi Tom!, 2023-07-19 16:22:11.557466, 2023-07-19 16:22:11.557466).
```
- Even when I tried using the data in your `seed/rb` file, it throws the same error. please, kindly fix this issue. :pray: 


## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ellyboi** [comment]
**Pictorial corrections made based on previous reviewer complain and suggestion**


codereview

File: db/migrate/20230719203740_add_body_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddBodyToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :body, :text
```
Comment (MahmoudBakr23) [inline db/migrate/20230719203740_add_body_to_comments.rb:3]: Optional
Please reconsider adding the body column to the comments table as it already has a text column. This causes to leave a body property set to nil on each record that's being created in the comment model.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Ellyboi 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the required changes from the previous reviewer
- Excellent communication with the reviewer

**Optional**
_See the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**